draft-ietf-quic-transport.md

```diff
@@ -3056,6 +3059,17 @@ a real ACK.
 A EMPTY_ACK frame has no content.  That is, a EMPTY_ACK frame consists of the
 single octet that identifies the frame as a EMPTY_ACK frame.
 
+## CRYPTO_CLOSE Frame {#frame-crypto-close}
+
+The CRYPTO_CLOSE frame (type=0x20) is used to indicate connection failures
```

Comment:
Are they always failures, or do you mean alerts? CONNECTION_CLOSE can be empty to signal graceful close.

Reply:
They are failures.